This job can be monitored from: https://job.c3se.chalmers.se/alvis/4587043
Filename (results):  /mimer/NOBACKUP/groups/brainage/thesis_brainage/results/proper_CV_w_MLP_wo_age/fold_0/val_fold.csv
We are looking at val data.
Model saved at thesis_brainage/results/proper_CV_w_MLP_wo_age/fold_0/model.pt will be applied to participants at /mimer/NOBACKUP/groups/brainage/thesis_brainage/results/proper_CV_w_MLP_wo_age/fold_0/val_fold.csv.
Last block has number of input channels: 64
Last block has number of output channels: 16
The model is CV_LILAC.
Traceback (most recent call last):
  File "/mimer/NOBACKUP/groups/brainage/thesis_brainage/scripts/apply_model.py", line 167, in <module>
    checkpoint = torch.load(args.model_state, map_location = device)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mimer/NOBACKUP/groups/brainage/thesis_brainage/my_venv/lib/python3.12/site-packages/torch/serialization.py", line 1425, in load
    with _open_file_like(f, "rb") as opened_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mimer/NOBACKUP/groups/brainage/thesis_brainage/my_venv/lib/python3.12/site-packages/torch/serialization.py", line 751, in _open_file_like
    return _open_file(name_or_buffer, mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mimer/NOBACKUP/groups/brainage/thesis_brainage/my_venv/lib/python3.12/site-packages/torch/serialization.py", line 732, in __init__
    super().__init__(open(name, mode))
                     ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'thesis_brainage/results/proper_CV_w_MLP_wo_age/fold_0/model.pt'
